docs/resources/monitor.md

```
@@ -761,3 +787,16 @@ Optional:
 
 - `packet_loss_threshold` (Number) Packet loss threshold percentage.
 - `payload` (String) Optional UDP payload to send.
+
+
+
+<a id="nestedatt--region_data"></a>
+### Nested Schema for `region_data`
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)